elements/pfe-progress-indicator/src/pfe-progress-indicator.scss

```diff
@@ -0,0 +1,68 @@
+@import "../../pfe-sass/pfe-sass";
+
+$pfe-progress-indicator--border-light: 4px solid rgba(0, 0, 0, 0.25);
+$pfe-progress-indicator--border-dark: 4px solid rgba(0, 0, 0, 0.75);
+
+:host([hidden]) {
+  display: none;
+}
+
+:host([pfe-indeterminate]) {
+    animation: spin 1s linear infinite;
+    -webkit-animation:spin 1s linear infinite;
+    -moz-animation:spin 1s linear infinite;
+
+    border-bottom: $pfe-progress-indicator--border-light;
+    border-left: $pfe-progress-indicator--border-light;
+    border-right: $pfe-progress-indicator--border-light;
+    border-top: $pfe-progress-indicator--border-dark;
+    border-radius: 100%;
+    
+    display: inline-block;
+    height: 2rem;
+    margin: 0 auto;
+    position: relative;
+    width: 2rem;
```

Comment:
Same here or use the same variable and name it something more generic like `--pfe-progress-indicator--size`
```
width: var(--pfe-progress-indicator--Width, 2rem);
```